basic-steps/src/main/java/org/jenkinsci/plugins/workflow/steps/ToolStep.java

```diff
@@ -80,7 +80,7 @@ public DescriptorImpl() {
         }
 
         @Override public String getDisplayName() {
-            return "Install a tool";
+            return "Use a tool";
```

Comment:
The `tool` step is not really installing the tool (it really depends on the existence of a `ToolInstaller`). That's why I think the term _Install_ here could be really confusing (actually, it was confusing to me before going to the code and see what the step execution really does).